networking_cisco/plugins/cisco/l3/drivers/asr1k/aci_asr1k_routertype_driver.py

```diff
@@ -0,0 +1,202 @@
+# Copyright 2015 Cisco Systems, Inc.  All rights reserved.
+#
+#    Licensed under the Apache License, Version 2.0 (the "License"); you may
+#    not use this file except in compliance with the License. You may obtain
+#    a copy of the License at
+#
+#         http://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+#    WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+#    License for the specific language governing permissions and limitations
+#    under the License.
+
+from networking_cisco.plugins.cisco.extensions import routerhostingdevice
+from networking_cisco.plugins.cisco.l3.drivers.asr1k import (
+    asr1k_routertype_driver as asr1k)
+from neutron.common import constants as l3_constants
+from neutron import context as nctx
+from neutron import manager
+
+from oslo_log import log as logging
+from oslo_utils import importutils
+
+LOG = logging.getLogger(__name__)
+
+
+APIC_OWNED = 'apic_owned_'
+UUID_REGEX = '[0-9a-f]{8}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{12}'
```

Comment:
I saw these 2 lines also defined in the other driver. Its better to move them to a single location if possible.